tests/console/login.go

```diff
@@ -77,7 +77,20 @@ func LoginToAlpine(vmi *v1.VirtualMachineInstance) error {
 	}
 	defer expecter.Close()
 
+	// Do not login, if we already logged in
 	b := append([]expect.Batcher{
+		&expect.BSnd{S: "\n"},
+		&expect.BExp{R: PromptExpression},
+		&expect.BSnd{S: "true\n"},
```

Comment:
What is the added value of sending `true`?

Reply:
same as https://github.com/kubevirt/kubevirt/pull/6116/files#r681193075